app/controllers/AmountsController.scala

```diff
@@ -4,10 +4,10 @@ import com.gu.googleauth.AuthAction
 import models.ConfiguredAmounts
 import models.ConfiguredAmounts._
 import play.api.mvc.{AnyContent, ControllerComponents}
-import zio.DefaultRuntime
+import zio.ZEnv
```

Comment:
In this PR I've updated the zio version, which unfortunately changes the `Runtime` definition slightly. So there are lots of import/type changes like this